pkg/kubelet/kuberuntime/kuberuntime_sandbox_linux.go

```diff
@@ -74,3 +75,36 @@ func (m *kubeGenericRuntimeManager) applySandboxResources(pod *v1.Pod, config *r
 
 	return nil
 }
+
+func (m *kubeGenericRuntimeManager) convertResourceConfigToLinuxContainerResources(rc *cm.ResourceConfig) *runtimeapi.LinuxContainerResources {
```

Comment:
I think a good test to add would be:

For each test pod:
1. calculate the linux sandbox resources & overhead (`calculateSandboxResources`, `convertOverheadToLinuxResources`)
2. calculate the pod resource config (`ResourceConfigForPod`)
3. generate the UpdatePodSandboxResources request (`generateUpdatePodSandboxResourcesRequest`)
4. verify that the resources & overhead from (1) match the resources & overhead from (3)

Reply:
Nice, done in TestGenerateUpdatePodSandboxResourcesRequest.